cli/package.json

```
@@ -13,7 +13,7 @@
     "lint": "bin-up eslint --fix *.js scripts/*.js bin/* lib/*.js lib/**/*.js test/*.js test/**/*.js",
     "prerelease": "npm run build",
     "release": "cd build && releaser --no-node --no-changelog",
-    "size": "t=\"$(npm pack .)\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";",
+    "size": "cross-env t=\"$(npm pack .)\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";",
```

so this command is no longer necessary, since new npm can
```suggestion
    "size": "npm pack --dry",
```

---

great, this command has always made me go :thinking:

---

yeah, it was not available in 2015 https://glebbahmutov.com/blog/smaller-published-NPM-modules/